File: dotnet/src/Model/World/GrassTile.cs
using System;

namespace PlanetCS1 {
    class GrassTile : RealTile {
        public GrassTile(Location location) {
            this.loc = location;
            this.occ = false;
            this.use = false;
            this.drive = false;
            this.traverse = true;
            this.owner = 0;
            this.icon = "G";
            this.minCost = 0;
            this.gasCost = 0;
            this.nrgCost = 0;
            this.health = 0;
            this.minSal = 0;
            this.gasSal = 0;
            this.nrgSal = 0;
        }
    }
}

File: jvm2/src/Model/World/GrassTile.kt
package Model.World

import Model.Abstract.RealTile
import Model.Location
import Model.Move

class GrassTile(location: Location): RealTile() {
    override var loc = location
    override var occ = false
    override var use = false
    override var drive = false
    override var traverse = true
    override var owner = 0
    override var icon = "G"
    override var minCost = 0
    override var gasCost = 0
    override var nrgCost = 0
    override var health = 0
    override var minSal = 0
    override var gasSal = 0
    override var nrgSal = 0
}